### ISSUE TITLE
Admin's `list_filter` not behaving correctly with EnumChoiceField

### ISSUE THREAD

**Issue by RadoRado** [HackSoftware/django-enum-choices#31]
If we add an `EnumChoiceField` in `list_filter`, then the URLs being generated looke like this:

```
status__exact=Enum.SOMETHING
```

which fails with

```
AttributeError:
'str' object has no attribute 'value'
```

**Comment by slavov-v** [comment]
[Django's docs for `list_filter`](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_filter) state that:

```
list_filter should be a list or tuple of elements, where each element should be of one of the following types:

- a field name, where the specified field should be either a BooleanField, 
CharField, DateField, DateTimeField, IntegerField, ForeignKey or ManyToManyField

- a class inheriting from django.contrib.admin.SimpleListFilter, which you need to provide
 the title and parameter_name attributes to and override the lookups and queryset methods
```
I'll have a look if we can export a custom list filter class for usage with `EnumChoiceField`

**Comment by RadoRado** [comment]
Closed by #34
